packages/next/shared/lib/head.tsx

```diff
@@ -161,11 +161,16 @@ function reduceComponents(
           return React.cloneElement(c, newProps)
         }
       }
-      // TODO(kara): warn for stylesheets as well as scripts
-      if (process.env.NODE_ENV === 'development' && c.type === 'script') {
-        console.warn(
-          `Do not add <script> tags using next/head. Use next/script instead. \nSee more info here: https://nextjs.org/docs/messages/no-script-tags-in-head-component`
-        )
+      if (process.env.NODE_ENV === 'development') {
+        if (c.type === 'script') {
+          console.warn(
+            `Do not add <script> tags using next/head. Use next/script instead. \nSee more info here: https://nextjs.org/docs/messages/no-script-tags-in-head-component`
+          )
+        } else if (c.type === 'link' && c.props['rel'] === 'stylesheet') {
+          console.warn(
+            `Do not add stylesheets using next/head. Use Document instead. \nSee more info here: https://nextjs.org/docs/messages/no-stylesheets-in-head-component`
```

Comment:
Should we log the `src` or `href` for these to help users track down where the warnings are coming from?

Reply:
Oh, that's a great idea. Can add that!

Reply:
Added!